scripts/docker/ndk_r25_compile.sh

```diff
@@ -17,23 +17,26 @@ cd /o-mvll/src
 mkdir -p build_ndk_r25 && cd build_ndk_r25
 cmake -GNinja .. \
       -DCMAKE_BUILD_TYPE=Release \
-      -DCMAKE_CXX_COMPILER=clang++-11 \
-      -DCMAKE_C_COMPILER=clang-11 \
+      -DCMAKE_CXX_COMPILER=clang++-14 \
+      -DCMAKE_C_COMPILER=clang-14 \
```

Comment:
Switching the host compiler to clang 14, because our target compiler is based on clang 14:
```
> llvm-toolchain-android/out/stage1-install/bin/clang --version | head -2
Android (stage1, based on r450784d1) clang version 14.0.7 (https://android.googlesource.com/toolchain/llvm-project 4c603efb0cca074e9238af8b4106c30add4418f6)
Target: x86_64-unknown-linux-gnu
```